### PR TITLE
Naming convention, adding data from website, and CsI test

### PR DESCRIPTION
**Pull request by aidanfitz**

### CODE REVIEW

**Review by parikshitbajpai — COMMENTED** [review]
Looks good. Do we have permission to share all the data files that are new?

File: README.md [hunk 1]
```diff
@@ -42,7 +42,14 @@ wsl --install
 ```
 If WSL fails to start properly after reboot, you may need to create virtual disk space for WSL following [these instructions](https://utf9k.net/blog/wsl2-vhd-issue/).
 
-Once the Ubuntu app is successfully installed, you can follow the [Building Thermochimica](#building-thermochimica) instructions below.
+When Ubuntu app is successfully installed, run the following.
+```bash
+sudo apt-get update
```
Comment (parikshitbajpai) [inline README.md:47]:
```suggestion
sudo apt update && sudo apt upgrade
```

File: python/pseudoBinaryPhaseDiagramGUI.py [hunk 2]
```diff
@@ -12,15 +12,15 @@
 phaseIncludeTol = 1e-8
 
 class CalculationWindow:
-    def __init__(self, parent, datafile, nElements, elements, active):
+    def __init__(self, parent, datafile, nElements, elements, active,scrollable=True):
```
Comment (parikshitbajpai) [inline python/pseudoBinaryPhaseDiagramGUI.py:15]:
```suggestion
    def __init__(self, parent, datafile, nElements, elements, active, scrollable=True):
```

File: python/pseudoBinaryPhaseDiagramGUI.py [hunk 3]
```diff
@@ -12,15 +12,15 @@
...
         self.parent = parent
         self.datafile = datafile
         self.nElements = nElements
         self.elements = elements
         self.active = active
         if self.active:
             self.makeLayout()
-            self.sgw = sg.Window(f'Phase Diagram Setup: {os.path.basename(self.datafile)}', self.layout, location = [400,0], finalize=True)
+            self.sgw = sg.Window(f'Phase Diagram Setup: {os.path.basename(self.datafile)}', self.layout,resizable=True,size=(600, 800),location = [400,0], finalize=True)
```
Comment (parikshitbajpai) [inline python/pseudoBinaryPhaseDiagramGUI.py:23]:
```suggestion
            self.sgw = sg.Window(f'Phase Diagram Setup: {os.path.basename(self.datafile)}', self.layout, resizable=True, size=(600,800), location = [400,0], finalize=True)
```

File: python/pseudoBinaryPhaseDiagramGUI.py [hunk 4]
```diff
@@ -254,8 +254,8 @@ def makeLayout(self):
         if (self.nElements < 8):
             elemLayout = [sg.Column(elem1Layout),sg.Column(elem2Layout)]
         else:
-            elemLayout = [sg.Column(elem1Layout,vertical_alignment='t', scrollable = True, vertical_scroll_only = True, expand_y = True),
-                          sg.Column(elem2Layout,vertical_alignment='t', scrollable = True, vertical_scroll_only = True, expand_y = True)]
+            elemLayout = [sg.Column(elem1Layout,vertical_alignment='t', scrollable = True, vertical_scroll_only = True,  size=(200, 100),expand_y = True),
```
Comment (parikshitbajpai) [inline python/pseudoBinaryPhaseDiagramGUI.py:257]:
```suggestion
            elemLayout = [sg.Column(elem1Layout, vertical_alignment='t', scrollable = True, vertical_scroll_only = True, size=(200,100), expand_y = True),
```

File: python/pseudoBinaryPhaseDiagramGUI.py [hunk 5]
```diff
@@ -254,8 +254,8 @@ def makeLayout(self):
...
+                          sg.Column(elem2Layout,vertical_alignment='t', scrollable = True, vertical_scroll_only = True, size=(200, 100), expand_y = True)]
```
Comment (parikshitbajpai) [inline python/pseudoBinaryPhaseDiagramGUI.py:258]:
```suggestion
                          sg.Column(elem2Layout, vertical_alignment='t', scrollable = True, vertical_scroll_only=True, size=(200,100), expand_y=True)]
```

**Review by parikshitbajpai — CHANGES_REQUESTED** [review]

**Comment by aidanfitz** [comment]
Sorry for the late response. I will get on it and make sure your suggestions will be used. I updated the file names to system-owner.dat. And all files are from the tutorial, excluding one that we cannot post there should be no need to worry on that end.

**Pull request merged**